tools/BUILD

```diff
@@ -66,10 +66,10 @@ sh_binary(
     name = "clang_format",
     srcs = ["format_cc.sh"],
     args = [
-        "$(rootpath @previous_llvm_toolchain//:clang-format)",
+        "$(rootpath @current_llvm_toolchain//:clang-format)",
```

Comment:
note: I believe these were the last use of `previous_llvm_toolchain`, but leaving it doesn't hurt.

Reply:
Yeah there is a manual switch in `.bazelrc` to switch back. I am planning on keeping around the ability to compile with the previous toolchain if someone wants to debug why something changed.